### ISSUE TITLE
Indentation guidelines

### ISSUE THREAD

**Issue by lenormf** [mawww/kakoune#3937]
### Feature 

A highlighter that displays indentation guidelines, similar to how [this plugin](https://github.com/nathanaelkane/vim-indent-guides) for Vim does it.

**Comment by Ordoviz** [comment]
duplicate of #2323

**Comment by lenormf** [comment]
Not quite, we already have indentation markers, but the request is specifically about guides, e.g. colouring the background differently — not inserting a character.

**Comment by andreyorst** [comment]
> Not quite, we already have indentation markers

IIUUC then #2323 can be closed? If not, then I don't see how this is not a duplicate, as whether to insert a character (it's not actually inserted, only displayed) or to display coloured background - it is just a different style of the same feature, that I've described in detail in #2323. I've also [proposed](https://github.com/mawww/kakoune/issues/2323#issuecomment-416163862) there that maybe there should be different styles of guides.
